src/plone/api/content.py

```diff
@@ -261,7 +261,10 @@ def copy(source=None, target=None, id=None, safe_id=False):
 
     new_id = copy_info[0]['new_id']
     if id:
-        return rename(obj=target[new_id], new_id=id, safe_id=safe_id)
+        if not safe_id and id in target:
+            raise InvalidParameterError
```

Comment:
Do we usually raise errors without any message in plone.api? If so this is ok and we should change it later.